combat-trainer.lic

```diff
@@ -171,13 +176,27 @@ class SetupProcess
       game_state.prepare_summoned_weapon(last_summoned)
     else
       @equipment_manager.wield_weapon(game_state.weapon_name, game_state.weapon_skill)
+      if game_state.use_whirlwind? && game_state.is_whirlwind_trainable?(game_state.weapon_skill)
```

Comment:
If `use_whirlwind` is only used in one place, it's a reasonable candidate for pulling out of GameState and making local to the process. This is not something we've been very consistent on... but GameState is pretty big already...